rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+    book.rent_books(self)
+    person.rent_books(self)
```

Comment:
- [ ] This should be `The rental` adding itself to the `@person` the 'belongs-to' association
`person.rentals << self`